R/run.R

```diff
@@ -319,3 +319,40 @@ view_run_details <- function(run) {
                 caption = paste(unlist(details$warnings), collapse = "\r\n"),
                 options = list(dom = "t", scrollY = TRUE))
 }
+
+#' Upload files to the run record.
+#' @param names The names of the files to upload. If set, paths must also be
```

Comment:
can you clarify what data structure user should use for `names` and `paths` parameters